### ISSUE TITLE
Documentation is a little scarce on how to run Prospector with additional dependencies in pre-commit hook. 

### ISSUE THREAD

**Issue by morgan-pubnub** [PyCQA/prospector#566]
This isn't a bug per se but a deficiency in the way the documentation is written with regards to actually getting the additional dependencies to work properly through the pre-commit hook. 

The documentation states:

This only installs base prospector - if you also use optional tools, for example bandit or mypy, then you can add them to the hook configuration like so:

```yaml
repos:
-   repo: https://github.com/PyCQA/prospector
    rev: 1.7.5
    hooks:
    -   id: prospector
        additional_dependencies:
        - ".[with_mypy,with_bandit]"
```

it should also include:

```yaml
repos:
-   repo: https://github.com/PyCQA/prospector
    rev: 1.7.5
    hooks:
    -   id: prospector
        additional_dependencies:
        - ".[with_mypy,with_bandit]"
       args: [
         '--with-tool=mypy',
         '--with-tool=bandit',
       ]
```

If you want to actually have `mypy` and `bandit` work properly. 

It wouldn't hurt to have an example of their configurations in the prospector.yml file as well. 

```yaml
# https://bandit.readthedocs.io/en/latest/config.html
bandit: 
  options:
    skips:
      - B201
      - B601
      - B610
      - B611
      - B703
  
# https://mypy.readthedocs.io/en/stable/command_line.html
mypy:
  options:
    ignore-missing-imports: true
```

**To Reproduce** Steps to reproduce the behavior:

Nothing to reproduce

I'm happy to make a PR with the appropriate modifications

**Comment by Pierre-Sassoulas** [comment]
LGTM, I'll merge a PR with the proposed change.

**Comment by Pierre-Sassoulas** [comment]
Closed by #571
